app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -111,4 +112,25 @@ def create_network_router_queue(userid, options = {})
     }
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
+
+  def create_floating_ip(options)
+    FloatingIp.raw_create_floating_ip(self, options)
```

Comment:
right, so the pattern we were doing was that the non raw method lives in the Floating IP class

so lets just keep this and we will refactor later